treeherder/model/tasks.py

```diff
@@ -132,7 +132,7 @@ def publish_job_action(project, action, job_id, requester):
 
 
 @task(name='publish-resultset-action')
-def publish_resultset_action(project, action, resultset_id, requester):
+def publish_resultset_action(project, action, resultset_id, requester, times=1):
     publisher = pulse_connection.get_publisher()
```

Comment:
publish_resultset_action is also used by trigger_missing_resultset_jobs() function. So added times=1 as a default parameter, so that the existing functions work.